src/main/java/domain/pages/HomePage.java

```diff
@@ -10,46 +12,53 @@
     Boolean result;
     String regex = "^[0-9]{5}(?:-[0-9]{4})?$";
 
-    @FindBy (xpath = "(//a[@href = '/representatives'])[1]")
+    @FindBy (xpath = "//a[@title='Home']")
+    private WebElement homePageLogo;
+
+    @FindBy(xpath = "(//a[@href = '/representatives'])[1]")
     private WebElement representativesLink;
 
-    @FindBy (xpath = "//button[@value='Look up'][1]")
+    @FindBy(xpath = "//button[@value='Look up'][1]")
     private WebElement lookUpZipCodeButton;
 
-    @FindBy (xpath = "//input[@id='header-find-rep-zip-code']")
+    @FindBy(xpath = "//input[@id='header-find-rep-zip-code']")
     private WebElement inputZipCode;
 
-    @FindBy (xpath = "//*[contains(text(), 'You submitted an invalid Zip Code')] ")
+    @FindBy(xpath = "//*[contains(text(), 'You submitted an invalid Zip Code')] ")
     private WebElement invalidZipCodeText;
 
-    @FindBy (xpath = "//img[@class='repPhoto'] ")
+    @FindBy(xpath = "//img[@class='repPhoto'] ")
     private WebElement representativePhoto;
 
-    @FindBy (xpath = "//div[@name='RepeatReps']")
+    @FindBy(xpath = "(//img[@class='repPhoto'])[2]")
     private WebElement atLeastTwoRepresentativesFound;
 
-    @FindBy (xpath = "//input[@value='FIND YOUR REP BY ADDRESS']")
+    @FindBy(xpath = "//input[@value='FIND YOUR REP BY ADDRESS']")
     private WebElement findYourRepByAddressButton;
 
-    public void clickRepresentativesLink(){
+    public void clickRepresentativesLink() {
         representativesLink.click();
     }
 
-    public String enterZipCode(String zipCode) {
-        Pattern pattern =Pattern.compile(regex);
-        Matcher matcher =pattern.matcher(zipCode);
+    public void enterZipCode(String zipCode) {
+        inputZipCode.clear();
         inputZipCode.sendKeys(zipCode);
         lookUpZipCodeButton.click();
```

Comment:
if you add this function into the implemnetation you should rename teh method as enterZipCodeAndSubmit

Reply:
corrected